cypress/integration/createNewUser.js

```diff
@@ -0,0 +1,32 @@
+/// <reference types="cypress" />
```

Comment:
This test is not working 
![image](https://user-images.githubusercontent.com/110588313/214792241-b7200427-1e2a-4b9a-89bf-27e8ec72b462.png)